content/blog/2022/2022-08-02-performance-with-completablefutures.md

```diff
@@ -0,0 +1,249 @@
+---
+title: "How to use CompletableFuture for better performance during API calls"
```

Comment:
We should remove the `during API calls` from the title, I think, because that narrows the article down too much. Maybe "How to use CompletableFuture to improve Performance"?

Reply:
done